version.props

```diff
@@ -0,0 +1,11 @@
+<!-- This file may be overwritten by automation. -->
+<Project>
+  <PropertyGroup>
+    <VersionPrefix>2.1.0</VersionPrefix>
```

Comment:
There isn't any reason to version artifacts from this repo. We shouldn't need this.

Reply:
If we remove all the version prefix stuff KoreBuild complains that the version `1.0.0` doesn't match `2.1.0-preview1-t000`. It doesn't really matter if the versions are "right" on this repo, but it doesn't hurt anything that they're there, so I don't see a need to circumvent KoreBuild.